package.json

```diff
@@ -67,11 +67,9 @@
     "image-size": "0.9.3",
     "imagemin": "7.0.1",
     "imagemin-gifsicle": "7.0.0",
-    "imagemin-jpegtran": "7.0.0",
     "imagemin-mozjpeg": "9.0.0",
     "imagemin-pngquant": "9.0.1",
     "imagemin-svgo": "8.0.0",
-    "include-media": "1.4.9",
```

Comment:
This for some helper functions in Sass https://eduardoboucas.github.io/include-media/, but there doesn't appear to be an references to `@include media` in the repo

Reply:
Any idea why we have this @schalkneethling The `yarn why` says:
```
▶ yarn why include-media
yarn why v1.22.10
[1/4] 🤔  Why do we have the module "include-media"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "include-media@1.4.9"
info Has been hoisted to "include-media"
info This module exists because it's specified in "dependencies".
info Disk size without dependencies: "144KB"
info Disk size with unique dependencies: "144KB"
info Disk size with transitive dependencies: "144KB"
info Number of shared dependencies: 0
```